test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorResolverTest.cs

```diff
@@ -1402,6 +1422,13 @@ private static TagHelperDescriptor CreatePrefixedStringDescriptor(string prefix)
                 assemblyName: stringType.GetTypeInfo().Assembly.GetName().Name);
         }
 
+        private static TagHelperDirectiveDescriptor CreateTagHelperDirectiveDescriptor(
+            string directiveText,
+            TagHelperDirectiveType directiveType)
+        {
+            return new TagHelperDirectiveDescriptor(directiveText, SourceLocation.Zero, directiveType);
+        }
```

Comment:
Realized this one doesn't even have an optional parameter. One argument less isn't much of an advantage over calling the constructor directly, even if you insist on not using the `internal` one.